pkg/kubectl/bind/plugin/bind.go

```diff
@@ -258,20 +264,10 @@ func (b *BindOptions) Run(ctx context.Context, urlCh chan<- string) error {
 
 	// print the request in dry-run mode
 	if b.DryRun {
-		printer, err := b.Print.ToPrinter()
-		if err != nil {
-			return err
-		}
-		first := true
 		for _, request := range apiRequests {
-			printer.PrintObj(request, b.IOStreams.Out) // nolint: errcheck
-
-			// TODO: this is a hack to separate the objects. Is there anything better in the printer?
-			if !first && *b.Print.OutputFormat == "yaml" {
-				fmt.Fprintln(b.IOStreams.Out, "---") // nolint: errcheck
+			if err = b.Printer.PrintObj(request, b.IOStreams.Out); err != nil {
```

Comment:
this is not good. It won't lead to a valid yaml file in case of yaml encoding, will it?

Reply:
I'll show in test cases that it is true :)